### PR TITLE
i18n(zh-cn): Create unhandled-rejection.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Create unhandled-rejection.mdx

#### Related issues & labels (optional)

- #6253

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/unhandled-rejection.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: 未处理的拒绝
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/unhandled-rejection.mdx:6]: delete this line

Comment (huyikai) [reply src/content/docs/zh-cn/reference/errors/unhandled-rejection.mdx]: Completed.

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**